CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - Scanner V4 may now be accessed anonymously for debugging purposes by enabling `ROX_SCANNER_V4_ALLOW_ANONYMOUS_AUTH`.
   - This was already enabled for development builds of StackRox, but now it may be configured for release builds, too.
   - This defaults to `true` for development builds, and `false` for release builds.
+- Deployment bundles created with roxctl do not contain PodSecurityPolicies (PSPs) anymore by default. When deploying to pre-1.25 Kubernetes clusters with PSPs enabled
```

Comment:
Nit, though I don't see any consistent line break structure in our readme, the line length here seems arbitrary - maybe recombine back to one line

Reply:
recombined.